src/ostorlab/cli/__init__.py

```diff
@@ -6,3 +6,5 @@
 from ostorlab.cli import agentgroup
 from ostorlab.cli import ci_scan
 from ostorlab.cli import scanner
+
+__all__ = [scan, auth, agent, vulnz, agentgroup, ci_scan, scanner]
```

Comment:
I would favor excluding the rule from ruff instead of adding `__all__` to the modules
this exporting method only affects the behavior of `from module import *` and is not used by us in anyway. plus the potential of future breakage.

fixing the error like this would just impose unused extra code.

Reply:
Nah, this is much better.